### ISSUE TITLE
Functionality to allow the usage of `Text` for axes ticks and labels in GraphScene.

### ISSUE THREAD

**Issue by AkashKarnatak** [ManimCommunity/manim#627]
It would be really nice if there was a way to use `Text` as axes ticks and labels instead of `Tex` in GraphScene as writing `Tex` takes a lot of time.

**Comment by leotrs** [comment]
Hello. Can you please provide an example of why "Tex takes a lot of time" and how you would like to see it fixed by using "Text"? Thanks.

**Comment by behackl** [comment]
I think this is a valid issue; the time it takes to render `Tex` is longer than the time required for rendering `Text`. 

As an example, a simple `GraphScene` like

```py
class FunctionPlot(GraphScene):
    CONFIG = {
        "x_min": -5,
        "x_max": 5,
        "x_labeled_nums": [-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5],
        "y_min": -0.5,
        "y_max": 2.5,
        "y_labeled_nums": [0.5, 1, 1.5, 2]
    }

    def construct(self):
        def func(x):
            return (x - int(x)) ** 2

        self.setup_axes()
        self.axes.move_to(ORIGIN)
        self.play(ShowCreation(self.axes))
        self.play(ShowCreation(self.get_graph(func)))
        self.wait()
``` 
takes on my system
```
9,68s user 3,93s system 175% cpu 7,749 total
```
Whereas, when rendering `DecimalNumber` as `Text` instead:
```
5,60s user 2,61s system 187% cpu 4,387 total
```

**Comment by leotrs** [comment]
Ah that makes sense. I didn't understand at first this was about runtime.

Agree this is a good idea. We have to come up with a good way of changing this though, otherwise we will have the same problem as in #462 and #380

**Comment by behackl** [comment]
Yes: we could in fact solve it in more or less the same way as #462 and #380 and allow passing rendered labels -- but in this case, we will have to be more careful as the values of the labels are also needed.
